docs/getting-started-guides/docker.md

```diff
@@ -30,12 +30,12 @@ Documentation for other releases can be found at
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-Running Kubernetes locally via Docker
```

Comment:
the capitalization changes were made by someone else -- github said my changes (which were non-conflicting) would merge fine...